src/descriptors-device.adb

```diff
@@ -379,7 +383,9 @@ package body Descriptors.Device is
                                  To_String (Device.Name),
                                  "Definition of the device's interrupts",
                                  False);
-         Add (Spec, New_With_Clause ("Ada.Interrupts", True));
+         if In_Runtime then
```

Comment:
This is never executed: it's within the else statement of "if In_Runtime".

Reply:
Right, I fixed this in the new version.

Thanks,